### PR TITLE
Minor fixes and improvements

### PR DESCRIPTION
**Pull request by Ori1N**
- added optional parameter kpi for convert call
- in android example api calls removed from main thread
- additional minor improvements

### CODE REVIEW

**Comment by sddamico** [comment]
@Ori1N Thanks for the contribution! I'm currently at the Google I/O conference and should be able to take a look at this early next week, thanks for your patience!

**Comment by Ori1N** [comment]
@sddamico Thanks! Looking forward to it!

File: example-android/src/main/java/com/seatgeek/sixpack/android/SixpackActivity.java [hunk 1]
```diff
@@ -23,6 +24,8 @@
 
     @InjectView(R.id.colorful_button) Button colorfulButton;
 
+    private ParticipatingExperiment mParticipatingExperiment = null;
```
Comment (sddamico) [inline example-android/src/main/java/com/seatgeek/sixpack/android/SixpackActivity.java]: nit: no hungarian notation please

File: sixpack-java/src/main/java/com/seatgeek/sixpack/ParticipatingExperiment.java [hunk 2]
```diff
@@ -36,7 +36,20 @@
...
+    }
+
+    /**
+     * Converts this experiment by notifying the Sixpack server that the user has completed the
+     * expected action
+     *
+     * This call makes blocking network requests.
+
+     * @param kpi The kpi of the conversion.
+     * @return
```
Comment (sddamico) [inline sixpack-java/src/main/java/com/seatgeek/sixpack/ParticipatingExperiment.java]: Please complete the documentation here about the return type and the potential to throw

**Comment by sddamico** [comment]
Just a couple small comments but otherwise this looks great!

**Comment by Ori1N** [comment]
Fixed the comments :)

**Comment by sddamico** [comment]
Thanks again!

**Pull request merged**